docs/install-service/setup-openshift-372.md

```diff
@@ -191,6 +191,11 @@ openshift_router_selector='region=infra'
 openshift_registry_selector='region=infra'
 openshift_disable_check=docker_storage,memory_availability
 
+# Openshift enable regular node cleanup. Note doing so may cause loss of logs (oc logs), but jenkins
+# logs will be retrieveable. Update values, if you think your cluster can handle more
+# Ref: https://docs.okd.io/latest/admin_guide/garbage_collection.html
```

Comment:
Since this is a OpenShift 3.7.2 doc, please link the upstream doc from 3.7.2 as well.

Reply:
https://docs.okd.io/3.7/admin_guide/garbage_collection.html